src/cascadia/TerminalApp/TabHeaderControl.cpp

```diff
@@ -18,8 +18,18 @@ namespace winrt::TerminalApp::implementation
         // We'll only process the KeyUp event if we received an initial KeyDown event first.
         // Avoids issue immediately closing the tab rename when we see the enter KeyUp event that was
         // sent to the command palette to trigger the openTabRenamer action in the first place.
-        HeaderRenamerTextBox().KeyDown([&](auto&&, auto&&) {
+        HeaderRenamerTextBox().KeyDown([&](auto&&, auto e) {
```

Comment:
```suggestion
        HeaderRenamerTextBox().KeyDown([&](auto&&, auto&& e) {
```

Prefer `auto&&` for lambda arguments; it will derive the most appropriate type :smile: